.eslint.log

```diff
@@ -1 +1,9 @@
 
+[4m/home/hiro/Documents/Projects/doccam-pi/node_modules/autossh/src/index.js[24m
```

Comment:
Looks like the lint is printing some warnings. Try to fix those this should be good to go. 👍

Reply:
True. Although some (if not all) were in already.

_Sent from my Sony D6603 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
Ok. I can fix the last three. But the first one was in already and requires significant changes...

_Sent from my Sony D6603 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
Sorry got busy again.
I'll check that out.